docs/deep-dive/scaffolding/your-first-scaffold-plugin.md

```diff
@@ -0,0 +1,135 @@
+---
+id: your-first-scaffold-plugin
+title: Your first scaffold plugin
```

Comment:
I think the title/sidebar_label should be changed to:
"Creating scaffold plugin"

Reply:
done